### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by stillsheryl**
This PR will:
- Add styling to make pages more readable
- Add spaces and margins
- Center text for readability
- All tests passing

Note: All email actions in users controller and friendships controller have been commented out. We need to add an email provider for the service to work on heroku. Tests have also been commented out.

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -30,17 +32,42 @@ footer {
...
+}
+
+.message {
+  padding-top: 25px;
+  padding-left: 25px;
+}
+
+#description {
+  text-align: center;
+}
```
Comment (Eternal-Flame085) [inline app/assets/stylesheets/application.css.scss:73]: Nice styling

**Review by Eternal-Flame085 — APPROVED** [review]
Nice!

**Comment by Stearnzy** [comment]
Build didn't pass! =(

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ def create
     if @user.save
       session[:user_id] = @user.id
 
-      send_welcome_email(@user)
+      # send_welcome_email(@user)
```
Comment (stillsheryl) [inline app/controllers/users_controller.rb:11]: As a group we decided to comment this code out so the app will run on heroku. We can go back and uncomment once we set up an email provider!

**Review by Stearnzy — APPROVED** [review]

**Pull request merged**